aptos-move/e2e-move-tests/src/tests/fee_payer.rs

```diff
@@ -0,0 +1,249 @@
+// Copyright © Aptos Foundation
```

Comment:
I think we need more tests for more cases such as:
1. Invalid gas payer signature
2. All valid signatures from senders and gas payers but signed over RawTransaction, now RawTransactionWithData
3. Gas payer account is a single signer (aka I pay for my own gas). I think most of the tests here only test when the payer is separate

Reply:
I can't find a way to test for invalid signatures. As everything is private or crate private and the only way to construct signatures are valid signatures

Reply:
https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos-crypto/src/ed25519/ed25519_sigs.rs#L43